--- a/lib/tls/CMakeLists.txt	2025-12-03 06:15:09.000000000 +0000
+++ b/lib/tls/CMakeLists.txt	2026-01-24 22:07:30.032318005 +0000
@@ -518,7 +518,7 @@
                execute_process(
                        COMMAND printf "GB\\nErewhon\\nAll around\\nlibwebsockets-test\\n\\nlocalhost\\nnone@invalid.org\\n"
                        COMMAND "${OPENSSL_EXECUTABLE}"
-                               req -config /etc/ssl/openssl.cnf -new -newkey rsa:2048 -days 10000 -nodes -x509 -keyout "${TEST_SERVER_SSL_KEY}" -out "${TEST_SERVER_SSL_CERT}"
+                               req -config /usr/share/defaults/ssl/openssl.cnf -new -newkey rsa:2048 -days 10000 -nodes -x509 -keyout "${TEST_SERVER_SSL_KEY}" -out "${TEST_SERVER_SSL_CERT}"
                        RESULT_VARIABLE OPENSSL_RETURN_CODE
                        #               OUTPUT_QUIET ERROR_QUIET
                        )
