From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Tue, 12 Aug 2025 15:41:24 -0500
Subject: [PATCH] Allow libdisplay-info 0.3

---
 libweston/backend-drm/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libweston/backend-drm/meson.build b/libweston/backend-drm/meson.build
index 002669e6..a8f2ea59 100644
--- a/libweston/backend-drm/meson.build
+++ b/libweston/backend-drm/meson.build
@@ -4,7 +4,7 @@ endif
 
 dep_libdisplay_info = dependency(
 	'libdisplay-info',
-	version: ['>= 0.1.1', '< 0.3.0'],
+	version: ['>= 0.1.1', '< 0.4.0'],
 	fallback: ['display-info', 'di_dep'],
 	default_options: [
 		'werror=false',
