diff --git a/src/effect/offscreenquickview.cpp b/src/effect/offscreenquickview.cpp
index 3dc83f9b2e..f9aa77c4ba 100644
--- a/src/effect/offscreenquickview.cpp
+++ b/src/effect/offscreenquickview.cpp
@@ -31,6 +31,7 @@
 #include <QQuickRenderTarget>
 #include <QTimer>
 #include <private/qeventpoint_p.h> // for QMutableEventPoint
+#include <private/qabstractanimation_p.h>
 
 namespace KWin
 {
@@ -214,6 +215,8 @@ void OffscreenQuickView::update()
         return;
     }
 
+    QUnifiedTimer::instance()->updateAnimationTimers();
+
     bool usingGl = d->m_glcontext != nullptr;
     OpenGlContext *previousContext = OpenGlContext::currentContext();
 
