From 7444be5834128ea026490bddde1537744da2a7e0 Mon Sep 17 00:00:00 2001
From: Aleksei Bavshin <alebastr89@gmail.com>
Date: Wed, 31 Jan 2024 08:57:06 -0800
Subject: [PATCH] contrib: fix typo in the portals config example

---
 contrib/wlroots-portals.conf | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/contrib/wlroots-portals.conf b/contrib/wlroots-portals.conf
index 0baa2ca4..0b4220fb 100644
--- a/contrib/wlroots-portals.conf
+++ b/contrib/wlroots-portals.conf
@@ -1,7 +1,7 @@
 [preferred]
 # Use xdg-desktop-portal-gtk for every portal interface...
 default=gtk
-# ... except for the Screencast, Screenshot and Settings (dark/light mode) interface
-org.freedesktop.impl.portal.Screencast=wlr
+# ... except for the ScreenCast, Screenshot and Settings (dark/light mode) interface
+org.freedesktop.impl.portal.ScreenCast=wlr
 org.freedesktop.impl.portal.Screenshot=wlr
 org.freedesktop.impl.portal.Settings=darkman
