From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Sat, 1 Nov 2025 15:57:41 -0500
Subject: [PATCH] icu 78 build fix

---
 intl/lwbrk/LineBreaker.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/intl/lwbrk/LineBreaker.cpp b/intl/lwbrk/LineBreaker.cpp
index 772170df8654..47578990ba44 100644
--- a/intl/lwbrk/LineBreaker.cpp
+++ b/intl/lwbrk/LineBreaker.cpp
@@ -454,6 +454,7 @@ static int8_t GetClass(uint32_t u, LineBreakRule aLevel,
       /* AKSARA_START = 45,                 [AS] */ CLASS_CHARACTER,
       /* VIRAMA_FINAL = 46,                 [VF] */ CLASS_CHARACTER,
       /* VIRAMA = 47,                       [VI] */ CLASS_CHARACTER,
+      /* UNAMBIGUOUS_HYPHEN = 47,                       [VI] */ CLASS_CHARACTER,
   };
 
   static_assert(U_LB_COUNT == std::size(sUnicodeLineBreakToClass),
