From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Fri, 19 Sep 2025 13:08:47 -0500
Subject: [PATCH] fix mandir

---
 doc/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/meson.build b/doc/meson.build
index 018b7d03..3fbbe4d3 100644
--- a/doc/meson.build
+++ b/doc/meson.build
@@ -43,7 +43,7 @@ if want_docs
         foreach type : types
             format = type[0]
             ext = type[1]
-            install_dir = type[2]
+            install_dir = type[2] + '/man1'
 
             xsl = 'http://docbook.sourceforge.net' + \
                   f'/release/xsl/current/@format@/docbook.xsl'
