From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Tue, 31 Jan 2023 20:37:52 -0600
Subject: [PATCH] Use serpent paths

---
 daemon/config/config_linux.go | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/daemon/config/config_linux.go b/daemon/config/config_linux.go
index 80c0ab8e49..0645949b5d 100644
--- a/daemon/config/config_linux.go
+++ b/daemon/config/config_linux.go
@@ -201,6 +201,9 @@ func lookupBinPath(binary string) (string, error) {
 	}
 
 	lookupPaths := []string{
+		// libexec on AerynOS/Solus
+		"/usr/lib/moby",
+
 		// FHS 3.0: "/usr/libexec includes internal binaries that are not intended to be executed directly by users or shell scripts. Applications may use a single subdirectory under /usr/libexec."
 		// https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch04s07.html
 		"/usr/local/libexec/docker",
