From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Tue, 26 Aug 2025 12:53:54 -0500
Subject: [PATCH] downstream: Update default PATH

---
 src/env/environment.rs | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/src/env/environment.rs b/src/env/environment.rs
index b818ab846..9d948aa01 100644
--- a/src/env/environment.rs
+++ b/src/env/environment.rs
@@ -570,9 +570,10 @@ fn setup_user(vars: &EnvStack) {
         colon_split(&[bytes2wcstring(cstr.to_bytes())])
     } else {
         vec![
-            WString::from_str(env!("PREFIX")) + L!("/bin"),
+            WString::from_str("/usr/local/bin"),
+            WString::from_str("/usr/local/sbin"),
             WString::from_str("/usr/bin"),
-            WString::from_str("/bin"),
+            WString::from_str("/usr/sbin"),
         ]
     };
     Box::leak(paths.into_boxed_slice())
