From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Wed, 19 Nov 2025 09:46:24 -0600
Subject: [PATCH] fix binary path

---
 resources/xwayland-satellite.service | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/resources/xwayland-satellite.service b/resources/xwayland-satellite.service
index 060ab6b..f68cefd 100644
--- a/resources/xwayland-satellite.service
+++ b/resources/xwayland-satellite.service
@@ -8,7 +8,7 @@ Requisite=graphical-session.target
 [Service]
 Type=notify
 NotifyAccess=all
-ExecStart=/usr/local/bin/xwayland-satellite
+ExecStart=/usr/bin/xwayland-satellite
 StandardOutput=journal
 
 [Install]
