From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Thu, 29 May 2025 22:58:20 -0500
Subject: [PATCH] No libatomic

---
 platform/linuxbsd/detect.py | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/platform/linuxbsd/detect.py b/platform/linuxbsd/detect.py
index b5e80f4a4d..8cca0c1351 100644
--- a/platform/linuxbsd/detect.py
+++ b/platform/linuxbsd/detect.py
@@ -496,11 +496,3 @@ def configure(env: "SConsEnvironment"):
     if platform.system() == "FreeBSD":
         env.Append(LINKFLAGS=["-lkvm"])
 
-    # Link those statically for portability
-    if env["use_static_cpp"]:
-        env.Append(LINKFLAGS=["-static-libgcc", "-static-libstdc++"])
-        if env["use_llvm"] and platform.system() != "FreeBSD":
-            env["LINKCOM"] = env["LINKCOM"] + " -l:libatomic.a"
-    else:
-        if env["use_llvm"] and platform.system() != "FreeBSD":
-            env.Append(LIBS=["atomic"])
