From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Sat, 6 Sep 2025 16:29:07 -0500
Subject: [PATCH] ffmpeg 7.1

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7f98c72..6c8028a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -49,7 +49,7 @@ find_package(KF6 ${KF_MIN_VERSION} REQUIRED
 )
 
 find_package(PkgConfig)
-pkg_check_modules(libavcodec REQUIRED IMPORTED_TARGET libavcodec)
+pkg_check_modules(libavcodec REQUIRED IMPORTED_TARGET libavcodec-7.1)
 
 configure_file(config.h.in ${CMAKE_CURRENT_BINARY_DIR}/config.h)
 
