From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Tue, 26 Aug 2025 13:59:08 -0500
Subject: [PATCH] downstream: Update default PATH

---
 build.rs | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/build.rs b/build.rs
index 3c4d6517..3bcfe480 100644
--- a/build.rs
+++ b/build.rs
@@ -18,12 +18,12 @@ fn main() {
         get_first_path(&["/var/mail", "/var/spool/mail", "/usr/spool/mail"]).unwrap_or("/var/mail");
 
     // TODO: use _PATH_STDPATH and _PATH_DEFPATH_ROOT from paths.h
-    println!("cargo:rustc-env=SUDO_PATH_DEFAULT=/usr/bin:/bin:/usr/sbin:/sbin");
+    println!("cargo:rustc-env=SUDO_PATH_DEFAULT=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin");
     println!(
-        "cargo:rustc-env=SU_PATH_DEFAULT_ROOT=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
+        "cargo:rustc-env=SU_PATH_DEFAULT_ROOT=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin"
     );
     println!(
-        "cargo:rustc-env=SU_PATH_DEFAULT=/usr/local/bin:/usr/bin:/bin:/usr/local/games:/usr/games"
+        "cargo:rustc-env=SU_PATH_DEFAULT=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin"
     );
 
     println!("cargo:rustc-env=PATH_MAILDIR={path_maildir}");
