diff -up expect-5.45.4.1/Makefile.in.1~ expect-5.45.4.1/Makefile.in
--- expect-5.45.4.1/Makefile.in.1~	2024-10-14 10:58:56.000000000 +0200
+++ expect-5.45.4.1/Makefile.in	2025-03-14 02:56:16.395756725 +0100
@@ -121,8 +121,8 @@ includedir	= @includedir@
 DESTDIR		=
 
 PKG_DIR		= $(PACKAGE_NAME)$(PACKAGE_VERSION)
-pkgdatadir	= $(datadir)/$(PKG_DIR)
-pkglibdir	= $(libdir)/$(PKG_DIR)
+pkgdatadir     = $(datadir)/tcl@TCL_VERSION@/$(PKG_DIR)
+pkglibdir      = $(libdir)/tcl@TCL_VERSION@/$(PKG_DIR)
 pkgincludedir	= $(includedir)/$(PKG_DIR)
 
 top_builddir	= .
@@ -263,7 +263,7 @@ install-doc: doc
 	  else true; fi ; \
 	done
 
-test: binaries libraries
+test: binaries libraries pkgIndex.tcl-test
 	$(TCLSH) `@CYGPATH@ $(srcdir)/tests/all.tcl` $(TESTFLAGS)
 
 shell: binaries libraries
@@ -331,6 +331,11 @@ pkgIndex.tcl:
 pkgIndex.tcl-hand:
 	(echo 'if {![package vsatisfies [package provide Tcl] @TCL_VERSION@]} {return}' ; \
 	 echo 'package ifneeded Expect $(PACKAGE_VERSION) \
+	    [list load [file join $$dir .. .. $(PKG_LIB_FILE).1]]'\
+	) > pkgIndex.tcl
+
+pkgIndex.tcl-test:
+	(echo 'package ifneeded Expect $(PACKAGE_VERSION) \
 	    [list load [file join $$dir $(PKG_LIB_FILE) ]]'\
 	) > pkgIndex.tcl
 
@@ -527,22 +532,22 @@ install-lib-binaries:
 	@mkdir -p $(DESTDIR)$(pkglibdir)
 	@list='$(lib_BINARIES)'; for p in $$list; do \
 	  if test -f $$p; then \
-	    echo " $(INSTALL_PROGRAM) $$p $(DESTDIR)$(pkglibdir)/$$p"; \
-	    $(INSTALL_PROGRAM) $$p $(DESTDIR)$(pkglibdir)/$$p; \
+	    echo " $(INSTALL_PROGRAM) $$p $(DESTDIR)$(libdir)/$$p"; \
+	    $(INSTALL_PROGRAM) $$p $(DESTDIR)$(libdir)/$$p; \
 	    stub=`echo $$p|sed -e "s/.*\(stub\).*/\1/"`; \
 	    if test "x$$stub" = "xstub"; then \
-		echo " $(RANLIB_STUB) $(DESTDIR)$(pkglibdir)/$$p"; \
-		$(RANLIB_STUB) $(DESTDIR)$(pkglibdir)/$$p; \
+		echo " $(RANLIB_STUB) $(DESTDIR)$(libdir)/$$p"; \
+		$(RANLIB_STUB) $(DESTDIR)$(libdir)/$$p; \
 	    else \
-		echo " $(RANLIB) $(DESTDIR)$(pkglibdir)/$$p"; \
-		$(RANLIB) $(DESTDIR)$(pkglibdir)/$$p; \
+		echo " $(RANLIB) $(DESTDIR)$(libdir)/$$p"; \
+		$(RANLIB) $(DESTDIR)$(libdir)/$$p; \
 	    fi; \
 	    ext=`echo $$p|sed -e "s/.*\.//"`; \
 	    if test "x$$ext" = "xdll"; then \
 		lib=`basename $$p|sed -e 's/.[^.]*$$//'`.lib; \
 		if test -f $$lib; then \
-		    echo " $(INSTALL_DATA) $$lib $(DESTDIR)$(pkglibdir)/$$lib"; \
-	            $(INSTALL_DATA) $$lib $(DESTDIR)$(pkglibdir)/$$lib; \
+		    echo " $(INSTALL_DATA) $$lib $(DESTDIR)$(libdir)/$$lib"; \
+	            $(INSTALL_DATA) $$lib $(DESTDIR)$(libdir)/$$lib; \
 		fi; \
 	    fi; \
 	  fi; \
diff -up expect-5.45.4.1/configure.in.1~ expect-5.45.4.1/configure.in
--- expect-5.45.4.1/configure.in.1~	2024-10-15 01:07:02.000000000 +0200
+++ expect-5.45.4.1/configure.in	2025-03-14 02:54:30.927907100 +0100
@@ -987,6 +987,7 @@ AC_SUBST(EXP_CC_SEARCH_FLAGS)
 AC_SUBST(SETUID)
 AC_SUBST(SETPGRP_VOID)
 AC_SUBST(DEFAULT_STTY_ARGS)
+AC_SUBST(TCL_VERSION)
 # Expect uses these from tclConfig.sh to make the main executable
 AC_SUBST(TCL_DL_LIBS)
 AC_SUBST(TCL_CC_SEARCH_FLAGS)
