diff -up expect-5.45.4.1/Makefile.in.omv~ expect-5.45.4.1/Makefile.in
diff -up expect-5.45.4.1/exp_chan.c.omv~ expect-5.45.4.1/exp_chan.c
--- expect-5.45.4.1/exp_chan.c.omv~	2025-03-14 03:18:31.691491465 +0100
+++ expect-5.45.4.1/exp_chan.c	2025-03-14 03:19:54.695327394 +0100
@@ -44,11 +44,11 @@ static int		ExpBlockModeProc (ClientData
 static int		ExpCloseProc (ClientData instanceData,
 			    Tcl_Interp *interp);
 static int		ExpClose2Proc (ClientData instanceData,
-			    Tcl_Interp *interp);
+			    Tcl_Interp *interp, int whatever);
 static int		ExpInputProc (ClientData instanceData,
 		            char *buf, int toRead, int *errorCode);
 static int		ExpOutputProc (
-			    ClientData instanceData, char *buf, int toWrite,
+			    ClientData instanceData, const char *buf, int toWrite,
                             int *errorCode);
 static void		ExpWatchProc (ClientData instanceData,
 		            int mask);
@@ -271,7 +271,7 @@ ExpInputProc(instanceData, buf, toRead,
 static int
 ExpOutputProc(instanceData, buf, toWrite, errorCodePtr)
     ClientData instanceData;		/* Exp state. */
-    char *buf;				/* The data buffer. */
+    const char *buf;				/* The data buffer. */
     int toWrite;			/* How many bytes to write? */
     int *errorCodePtr;			/* Where to store error code. */
 {
@@ -379,9 +379,10 @@ ExpCloseProc(instanceData, interp)
 }
 /*ARGSUSED*/
 static int
-ExpClose2Proc(instanceData, interp)
+ExpClose2Proc(instanceData, interp, whatever)
     ClientData instanceData;	/* Exp state. */
     Tcl_Interp *interp;		/* For error reporting - unused. */
+    int whatever; /* whatever - unused */
 {
     ExpState *esPtr = (ExpState *) instanceData;
     ExpState **nextPtrPtr;
diff -up expect-5.45.4.1/exp_command.c.omv~ expect-5.45.4.1/exp_command.c
--- expect-5.45.4.1/exp_command.c.omv~	2025-03-14 03:08:12.629735056 +0100
+++ expect-5.45.4.1/exp_command.c	2025-03-14 03:08:40.962634287 +0100
@@ -1962,7 +1962,7 @@ static int
 Exp_SendObjCmd(
     ClientData clientData,
     Tcl_Interp *interp,
-    Tcl_Size objc,
+    int objc,
     Tcl_Obj *const objv[])
 {
     ThreadSpecificData *tsdPtr = TCL_TSD_INIT(&dataKey);
@@ -2475,7 +2475,7 @@ static int
 Exp_ExitObjCmd(
     ClientData clientData,
     Tcl_Interp *interp,
-    Tcl_Size objc,
+    int objc,
     Tcl_Obj *const objv[])		/* Argument objects. */
 {
     int value = 0;
diff -up expect-5.45.4.1/exp_int.h.omv~ expect-5.45.4.1/exp_int.h
--- expect-5.45.4.1/exp_int.h.omv~	2025-03-14 03:17:38.908950554 +0100
+++ expect-5.45.4.1/exp_int.h	2025-03-14 03:17:47.989100618 +0100
@@ -27,7 +27,7 @@ void	expDiagLogPtr       (char *);
 void	expDiagLogPtrX      (char *,int);
 void	expDiagLogPtrStr    (char *,char *);
 void	expDiagLogPtrStrStr (char *,char *,char *);
-void	expErrnoMsgSet      (char * (*) (int));
+void	expErrnoMsgSet      (const char * (*) (int));
 char * expErrnoMsg    (int);
 
 #ifdef NO_STDLIB_H
diff -up expect-5.45.4.1/exp_inter.c.omv~ expect-5.45.4.1/exp_inter.c
--- expect-5.45.4.1/exp_inter.c.omv~	2025-03-14 03:15:26.618974155 +0100
+++ expect-5.45.4.1/exp_inter.c	2025-03-14 03:15:58.843136327 +0100
@@ -696,8 +696,8 @@ static char *
 inter_updateproc(
     ClientData clientData,
     Tcl_Interp *interp,	/* Interpreter containing variable. */
-    char *name1,	/* Name of variable. */
-    char *name2,	/* Second part of variable name. */
+    const char *name1,	/* Name of variable. */
+    const char *name2,	/* Second part of variable name. */
     int flags)		/* Information about what happened. */
 {
 	exp_configure_count++;
@@ -714,7 +714,7 @@ int
 Exp_InteractObjCmd(
     ClientData clientData,
     Tcl_Interp *interp,
-    Tcl_Size objc,
+    int objc,
     Tcl_Obj *const initial_objv[])		/* Argument objects. */
 {
     ThreadSpecificData *tsdPtr = TCL_TSD_INIT(&dataKey);
diff -up expect-5.45.4.1/exp_main_sub.c.omv~ expect-5.45.4.1/exp_main_sub.c
diff -up expect-5.45.4.1/exp_tty.c.omv~ expect-5.45.4.1/exp_tty.c
--- expect-5.45.4.1/exp_tty.c.omv~	2025-03-14 03:16:08.312582619 +0100
+++ expect-5.45.4.1/exp_tty.c	2025-03-14 03:16:38.825606931 +0100
@@ -374,7 +374,7 @@ Exp_SttyCmd(
     ClientData clientData,
     Tcl_Interp *interp,
     int argc,
-    char **argv)
+    const char **argv)
 {
 	/* redirection symbol is not counted as a stty arg in terms */
 	/* of recognition. */
@@ -573,7 +573,7 @@ Exp_SystemCmd(
     ClientData clientData,
     Tcl_Interp *interp,
     int argc,
-    char **argv)
+    const char **argv)
 {
 	int result = TCL_OK;
 	RETSIGTYPE (*old)();	/* save old sigalarm handler */
diff -up expect-5.45.4.1/expect.c.omv~ expect-5.45.4.1/expect.c
--- expect-5.45.4.1/expect.c.omv~	2025-03-14 03:09:42.737777103 +0100
+++ expect-5.45.4.1/expect.c	2025-03-14 03:15:11.318851672 +0100
@@ -163,8 +163,8 @@ static char *exp_indirect_update1( /* 1-
 static char *exp_indirect_update2( /* 2-part Tcl variable names */
     ClientData clientData,
     Tcl_Interp *interp,	/* Interpreter containing variable. */
-    char *name1,	/* Name of variable. */
-    char *name2,	/* Second part of variable name. */
+    const char *name1,	/* Name of variable. */
+    const char *name2,	/* Second part of variable name. */
     int flags);		/* Information about what happened. */
 
 #ifdef SIMPLE_EVENT
@@ -324,8 +324,9 @@ exp_eval_with_one_arg(
 	    for (tokenPtr = parse.tokenPtr; numWords > 0;
 		 numWords--, tokenPtr += (tokenPtr->numComponents + 1)) {
 		/* FUTURE: Save token information, do substitution later */
-		Tcl_Obj* w = Tcl_EvalTokensStandard(interp, tokenPtr+1,
+		int r = Tcl_EvalTokensStandard(interp, tokenPtr+1,
 			tokenPtr->numComponents); 
+		Tcl_Obj* w = (r == TCL_OK) ? Tcl_GetObjResult(interp) : NULL;
 
 		/* w has refCount 1 here, if not NULL */
 		if (w == NULL) {
@@ -1384,7 +1385,7 @@ int
 Exp_ExpectGlobalObjCmd(
     ClientData clientData,
     Tcl_Interp *interp,
-    Tcl_Size objc,
+    int objc,
     Tcl_Obj *const objv[])		/* Argument objects. */
 {
     int result = TCL_OK;
@@ -2121,8 +2122,8 @@ static char *
 exp_indirect_update2(
     ClientData clientData,
     Tcl_Interp *interp,	/* Interpreter containing variable. */
-    char *name1,	/* Name of variable. */
-    char *name2,	/* Second part of variable name. */
+    const char *name1,	/* Name of variable. */
+    const char *name2,	/* Second part of variable name. */
     int flags)		/* Information about what happened. */
 {
 	char *msg;
@@ -2530,7 +2531,7 @@ int
 Exp_ExpectObjCmd(
     ClientData clientData,
     Tcl_Interp *interp,
-    Tcl_Size objc,
+    int objc,
     Tcl_Obj *const objv[])		/* Argument objects. */
 {
     int cc;			/* number of chars returned in a single read */
diff -up expect-5.45.4.1/pty_termios.c.omv~ expect-5.45.4.1/pty_termios.c
--- expect-5.45.4.1/pty_termios.c.omv~	2025-03-14 03:20:10.353391292 +0100
+++ expect-5.45.4.1/pty_termios.c	2025-03-14 03:20:44.236500393 +0100
@@ -9,6 +9,9 @@ would appreciate credit if you use this
 
 #include <stdio.h>
 #include <signal.h>
+#include <pty.h>
+
+extern char * expErrnoMsg    (int);
 
 #if defined(SIGCLD) && !defined(SIGCHLD)
 #define SIGCHLD SIGCLD
