From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Tue, 15 Jul 2025 23:11:56 -0500
Subject: [PATCH] fix build with econf

---
 login-utils/meson.build | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/login-utils/meson.build b/login-utils/meson.build
index 251d8de..87be4b9 100644
--- a/login-utils/meson.build
+++ b/login-utils/meson.build
@@ -39,6 +39,10 @@ if lib_selinux.found()
   chfn_chsh_deps += [lib_selinux]
 endif
 
+if lib_econf.found()
+  chfn_chsh_deps += [lib_econf]
+endif
+
 chfn_sources = files(
   'chfn.c',
 )
